packages/@okta/vuepress-site/docs/concepts/event-hooks/index.md

```diff
@@ -93,6 +93,15 @@ As a best practice, you should return the HTTP response immediately, rather than
 
 Event Hooks are limited to sending 100,000 events per 24-hour period. 
 
+### Debugging
+
+Events identified for delivery to your event hooks contain information about which event hooks were attempted for delivery.
+The `debugData` section of the [LogEvent](/docs/reference/api/system-log/#example-logevent-object) object contains the IDs of the event hooks that the particular event was delivered to.
+
+Note that this information is available in the event regardless of whether the delivery was successful or failed.
+
+Thus, in conjunction with the `event_hook.delivery` System Log event, you can debug an end-to-end flow.
```

Comment:
A minor addition to this sentence:
**"The event_hook.delivery event is produced only when event delivery fails."** 

-------- 
We need to clarify this because some early adopters were expecting to see an event_hook.delivery event during successful deliveries as well.